EventsExpress/ClientApp/src/components/login/login.js

```diff
@@ -40,6 +41,7 @@ class Login extends Component {
               </Button >
               <Button fullWidth={true} type="submit" value="Login" color="primary">
                 Sign In
+                <Redirect to='/home' />
```

Comment:
Why do we need `Redirect` here?

Reply:
Because if you will login after you got unauthorized you will logged in  on unauthorized page which makes no sense